portfolio/src/main/webapp/style.css

```diff
@@ -2,16 +2,17 @@
     font-size: 16px;
     font-family:'Roboto';
  
-    /* Color variables */
+    /* Variables */
```

Comment:
I like how you set these colors as variables so you don't have to repeat them throughout the file!